src/express/common/config-utils.js

```diff
@@ -745,9 +841,11 @@ export async function editMaticCliDockerYAMLConfig() {
   )
 }
 
-export async function validateConfigs() {
-  validateEnvVars()
-  validateAwsKeyAndCertificate()
+export async function validateConfigs(cloud) {
+  validateEnvVars(cloud)
+  if (cloud === 'aws') {
+    validateAwsKeyAndCertificate()
```

Comment:
Isn't this used at all for `gcp`? We could still keep the name and path of the key previously generated with `ssh-keygen`

Reply:
In GCP we do not have keypair name. Both public and private keys should exist on local